webtau-graphql/src/main/java/org/testingisdocumenting/webtau/graphql/GraphQLConfig.java

```diff
@@ -16,19 +16,26 @@
 
 package org.testingisdocumenting.webtau.graphql;
 
+import static org.testingisdocumenting.webtau.cfg.ConfigValue.declare;
+
+import java.util.stream.Stream;
 import org.testingisdocumenting.webtau.cfg.ConfigValue;
 import org.testingisdocumenting.webtau.cfg.WebTauConfig;
 import org.testingisdocumenting.webtau.cfg.WebTauConfigHandler;
 
-import java.util.stream.Stream;
-
-import static org.testingisdocumenting.webtau.cfg.ConfigValue.declare;
-
 public class GraphQLConfig implements WebTauConfigHandler {
     static final ConfigValue ignoreIntrospectionFailures = declare(
             "graphQLIgnoreIntrospectionFailures",
             "ignore graphQL introspection failures, introspection is required for coverage reporting",
             () -> true);
+    static final ConfigValue graphqlEndpoint = declare(
+            "graphqlEndpoint",
+            "override the default GraphQL endpoint",
+            () -> GraphQL.GRAPHQL_URL);
+    static final ConfigValue showGraphqlOperationAsQueryParam = declare(
+            "showGraphqlOperationAsQueryParam",
```

Comment:
should we use graphQL prefix for all the graphql options? QL is used in the option above instead of lower case ql.